### PR TITLE
renaming apex project to nexodus

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
Project apex has been renamed to "nexodus" updating all references in rosa-apps.
/cc @cooktheryan 
/cc @vishnoianil

### CODE REVIEW

File: argocd/overlays/rosa/applications/envs/rosa/nexodus/apex-qa.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: nexodus-qa
+spec:
+  destination:
+    name: in-cluster
+    namespace: nexodus-qa
+  source:
+    path: deploy/apex/overlays/qa
```
Comment (vishnoianil) [inline argocd/overlays/rosa/applications/envs/rosa/nexodus/apex-qa.yaml:10]: `deploy/apex/overlays/qa` is now `deploy/nexodus/overlays/qa

File: argocd/overlays/rosa/applications/envs/rosa/nexodus/apex-qa.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    repoURL: "https://github.com/redhat-et/apex"
```
Comment (vishnoianil) [inline argocd/overlays/rosa/applications/envs/rosa/nexodus/apex-qa.yaml:11]: repo url moved to `https://github.com/nexodus-io/nexodus`

File: argocd/overlays/rosa/applications/envs/rosa/nexodus/apex.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+kind: Application
+metadata:
+  name: nexodus
+spec:
+  destination:
+    name: in-cluster
+    namespace: nexodus
+  source:
+    path: deploy/nexodus/overlays/prod
+    repoURL: 'https://github.com/redhat-et/apex'
```
Comment (vishnoianil) [inline argocd/overlays/rosa/applications/envs/rosa/nexodus/apex.yaml:11]: same here, url moved to `https://github.com/nexodus-io/nexodus`

File: argocd/overlays/rosa/applications/envs/rosa/nexodus/apex-qa.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+apiVersion: argoproj.io/v1alpha1
```
Comment (vishnoianil) [inline argocd/overlays/rosa/applications/envs/rosa/nexodus/apex-qa.yaml:1]: I think it's good if we can change the name of file from apex-qa.yaml to nexodus-qa.yaml

File: argocd/overlays/rosa/applications/envs/rosa/nexodus/apex.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1alpha1
```
Comment (vishnoianil) [inline argocd/overlays/rosa/applications/envs/rosa/nexodus/apex.yaml:1]: we can rename this to nexodus.yaml

File: cluster-scope/base/core/namespaces/nexodus-monitoring/namespace.yaml [hunk 6]
```diff
@@ -5,6 +5,6 @@ metadata:
     op1st/docs: https://github.com/redhat-et/apex
```
Comment (vishnoianil) [inline cluster-scope/base/core/namespaces/nexodus-monitoring/namespace.yaml:5]: repo url needs an update.

File: cluster-scope/base/core/namespaces/nexodus-qa/namespace.yaml [hunk 7]
```diff
@@ -5,6 +5,6 @@ metadata:
     op1st/docs: https://github.com/redhat-et/apex
```
Comment (vishnoianil) [inline cluster-scope/base/core/namespaces/nexodus-qa/namespace.yaml:5]: url needs a fix.

File: cluster-scope/base/core/namespaces/nexodus/namespace.yaml [hunk 8]
```diff
@@ -5,9 +5,9 @@ metadata:
     op1st/docs: https://github.com/redhat-et/apex
```
Comment (vishnoianil) [inline cluster-scope/base/core/namespaces/nexodus/namespace.yaml:5]: url fixing.

File: cluster-scope/base/user.openshift.io/groups/nexodus/group.yaml [hunk 9]
```diff
@@ -1,7 +1,7 @@
 apiVersion: user.openshift.io/v1
 kind: Group
 metadata:
-  name: apex
+  name: nexodus
 users:
   - russellb
   - nerdalert
```
Comment (vishnoianil) [inline cluster-scope/base/user.openshift.io/groups/nexodus/group.yaml:7]: can you please add me to the users as well `avishnoi`

File: cluster-scope/overlays/rosa/kustomization.yaml [hunk 10]
```diff
@@ -16,16 +16,16 @@ resources:
   # --------------------------------------------------------------------------------------
   # Namespaces
   # --------------------------------------------------------------------------------------
-  - ../../base/core/namespaces/apex
-  - ../../base/core/namespaces/apex-monitoring
-  - ../../base/core/namespaces/apex-qa
+  - ../../base/core/namespaces/nexodus-monitoring
```
Comment (vishnoianil) [inline cluster-scope/overlays/rosa/kustomization.yaml:19]: I think it should be `nexodus` only.

File: cluster-scope/overlays/rosa/kustomization.yaml [hunk 11]
```diff
@@ -16,16 +16,16 @@ resources:
...
+  - ../../base/core/namespaces/nexodus-monitoring-monitoring
```
Comment (vishnoianil) [inline cluster-scope/overlays/rosa/kustomization.yaml:20]: It should be nexodus-monitoring?

File: cluster-scope/overlays/rosa/kustomization.yaml [hunk 12]
```diff
@@ -16,16 +16,16 @@ resources:
...
+  - ../../base/core/namespaces/nexodus-monitoring-qa
```
Comment (vishnoianil) [inline cluster-scope/overlays/rosa/kustomization.yaml:21]: it should be `nexodus-qa`

File: cluster-scope/overlays/rosa/kustomization.yaml [hunk 13]
```diff
@@ -16,16 +16,16 @@ resources:
...
   - ../../base/core/namespaces/copilot-ops
   - ../../base/core/namespaces/dex
   - ../../base/core/namespaces/ipfs
   # --------------------------------------------------------------------------------------
   # Operatorgroups and subscriptions
   # --------------------------------------------------------------------------------------
-  - ../../base/operators.coreos.com/operatorgroups/apex
+  - ../../base/operators.coreos.com/operatorgroups/nexodus-monitoring
```
Comment (vishnoianil) [inline cluster-scope/overlays/rosa/kustomization.yaml:28]: it should be nexodus only.

**Review by vishnoianil — CHANGES_REQUESTED** [review]

File: cluster-scope/base/core/namespaces/nexodus/kustomization.yaml [hunk 14]
```diff
@@ -2,8 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```
Comment (vishnoianil) [inline cluster-scope/base/core/namespaces/nexodus/kustomization.yaml:2]: Just assuming that this file should be moved from apex-qa to nexodus-qa?

File: cluster-scope/base/core/namespaces/nexodus/kustomization.yaml [hunk 15]
```diff
@@ -2,8 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
 resources:
 - namespace.yaml
-- ../../../../components/project-admin-rolebindings/apex
-namespace: apex-qa
+- ../../../../components/project-admin-rolebindings/nexodus
+namespace: nexodus
```
Comment (vishnoianil) [inline cluster-scope/base/core/namespaces/nexodus/kustomization.yaml:6]: the namespace should be nexodus-qa?

File: cluster-scope/base/operators.coreos.com/operatorgroups/nexodus/operatorgroup.yaml [hunk 16]
```diff
@@ -2,7 +2,9 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
-  name: apex
+  name: nexodus
 spec:
   targetNamespaces:
-    - apex
+    - nexodus
+    - nexodus-qa
```
Comment (vishnoianil) [inline cluster-scope/base/operators.coreos.com/operatorgroups/nexodus/operatorgroup.yaml:9]: do we need -qa and -monitoring namespace also listed as a a targetNamespaces here?

File: cluster-scope/base/user.openshift.io/groups/nexodus/group.yaml (hunk 9 above)
Comment (vishnoianil) [reply cluster-scope/base/user.openshift.io/groups/nexodus/group.yaml:7]: Thanks @Gregory-Pereira

**Review by vishnoianil — CHANGES_REQUESTED** [review]

File: cluster-scope/base/core/namespaces/nexodus/kustomization.yaml (hunk 14 above)
Comment (Gregory-Pereira) [reply cluster-scope/base/core/namespaces/nexodus/kustomization.yaml:2]: Git does some weird things when it comes to renaming which files to what when changes like this happen. All 3 namespaces exist, even if its changing the old prod one to current qa

File: cluster-scope/base/operators.coreos.com/operatorgroups/nexodus/operatorgroup.yaml (hunk 16 above)
Comment (Gregory-Pereira) [reply cluster-scope/base/operators.coreos.com/operatorgroups/nexodus/operatorgroup.yaml:9]: I believe we will need to add it as it is added here.

Comment (vishnoianil) [reply cluster-scope/base/operators.coreos.com/operatorgroups/nexodus/operatorgroup.yaml:9]: It was not added in the apex version of it, so I was not sure of it.

**Pull request merged**